README.md

```diff
@@ -35,22 +35,23 @@ cd cerberus_anymal_locomotion/
 git submodule update --init --recursive
 ```
 
-Configure and build the workspace. We need to overlay the `subt_ws` workspace because of protobuf version mismatch between one of our dependency (tensorflow-cpp) and the version used by Ignition Gazebo.
+Configure and build the workspace. We need to overlay the `subt_ws` workspace because of protobuf version mismatch between one of our dependency (tensorflow-cpp) and the version used by Ignition Gazebo. Furthermore we build in `release` mode.
 ```
 source /opt/ros/melodic/setup.bash
 
 cd ~/anymal_locomotion_ws/
 
 catkin config --extend ~/subt_ws/install/
+catkin config --cmake-args -DCMAKE_BUILD_TYPE=release
```

Comment:
:+1:

Reply:
Isn't it supposed to be "Release" or does capitalization not matter?

Reply:
Apparently everybody uses Release. The brahma settings from AB use release. I guess i does not matter but could not find any proof in this short search.

Reply:
The [official cmake documentation](https://cmake.org/cmake/help/v3.0/variable/CMAKE_BUILD_TYPE.html) says "Release". I also found anecdotes online that it doesn't matter, but I would stick to the official guidlines.